lib/index.js

```diff
@@ -71,6 +72,29 @@ class JsonEditor {
         return this.toObject();
     }
 
+    /**
+     * unset
+     * Remove a path from a JSON object.
+     *
+     * @name unset
+     * @function
+     * @param {String} path The object path.
+     * @returns {JsonEditor} The `JsonEditor` instance.
+     */
+    unset (path) {
+        if (typeof path === "object") {
```

Comment:
I'd rather say to use `this.set(path, undefined)` instead. Because `undefined` is not serialized in JSON.stringify.

Reply:
all set